web/src/enterprise/campaigns/detail/CampaignDetails.tsx

```diff
@@ -115,6 +115,8 @@ export const CampaignDetails: React.FunctionComponent<Props> = ({
         if (!campaignID) {
             return
         }
+        // on the very first fetch, a reload of the changesets is not required
+        let isFirstFetch = true
```

Comment:
Since there are multiple fetches going on in this file ⬇️ 
```suggestion
        let isFirstCampaignFetch = true
```